st_162h.nv.json

```diff
@@ -1,15 +1,15 @@
 {
   "_notes": [
-    "Compiled by NV Mapping Helper (Replace with your name)"
+    "Compiled by Tom Collins, based on st_162h by HorsePin"
   ],
-  "_copyright": "Copyright (C) 2022 by YourName your email>",
+  "_copyright": "Copyright (C)2024 by Tom Collins <tom@tomlogic.com>",
   "_license": "GNU Lesser General Public License v3.0",
   "_endian": "little",
   "_ramsize": 131118,
   "_roms": [
     "st_162h"
   ],
-  "_fileformat": 0.1,
+  "_fileformat": 0.2,
```

Comment:
should this be a new format?

Reply:
- [ ] Update `_fileformat` to 0.3 due to `null` property.
- [ ] Update normalize-map to check for `null` property.